.github/workflows/ci.yml

```diff
@@ -34,7 +34,19 @@ jobs:
           override: true
 
       # Build caching action
-      - uses: Swatinem/rust-cache@v1
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock', '**/Cargo.toml') }}
+            ${{ runner.os }}-cargo-
```

Comment:
Could you elaborate a little on what these changes do? Were we having some kind of issue with `rust-cache` in CI?